cmd/retrolabeler/main.go

```diff
@@ -0,0 +1,332 @@
+package main
```

Comment:
You could put the file in the root directory instead of nesting into `cmd`, which will simplify the install command as well.